tasks/managed/add-fbc-contribution/add-fbc-contribution.yaml

```
@@ -616,16 +616,23 @@ spec:
 
         # Deduplicate results: keep only the last component per unique target_index (the last one)
         # This ensures downstream tasks (collect-index-images, create-pyxis-image) use the correct digest
-        UNIQUE_TARGETS=$(jq -r '[.components[].target_index] | unique | length' "$RESULTS_FILE")
+        # if .target_index is empty, then use .ocp_version instead
```

```
       # Deduplicate results: keep only the last component per unique target_index (the last one)
       # This ensures downstream tasks (collect-index-images, create-pyxis-image) use the correct digest
       # if .target_index is empty, then use .ocp_version instead
```
This doesn't read great - there is no period or comma at the end of line 618, so it reads like: "... use the correct digest if .target_index is empty, then use .ocp_version instead". You see what I mean?


```suggestion
       # Deduplicate results: Keep only the last component per unique target_index (the last one).
       # This ensures downstream tasks (collect-index-images, create-pyxis-image) use the correct digest.
       # If .target_index is empty, then use .ocp_version instead
```

---

I see. It reads like a continuation of the previous line. I will change.

---

I rewrote the phrase as I also rewrote the code to make the logic more explicit.